kf/bpfmap.go

```diff
@@ -108,12 +111,18 @@ func (b *MetricsBPFMap) GetValue() float64 {
 		// We have observed in smaller configuration VM's, if we restart KF's
 		// Stale mapID's are reported, in such cases re-checking map id
 		log.Warn().Err(err).Msgf("GetValue : NewMapFromID failed ID %d, re-looking up of map id", b.MapID)
-		tmpBPF, err := b.BPFProg.GetBPFMap(b.Name)
+		tmp, err := b.BPFProg.GetBPFMap(b.Name)
 		if err != nil {
-			log.Warn().Err(err).Msgf("GetValue: Update new map ID %d", tmpBPF.MapID)
+			log.Warn().Err(err).Msgf("GetValue: Update new map ID")
```

Comment:
Why remove the old MapID from the log?

Reply:
This needs to be undone. Thanks for pointing this out